.gitignore

```diff
@@ -11,3 +11,4 @@
 
 # Other
 *.log
+.*.swp
```

Comment:
Tip: Put this type of thing in `~/.config/git/ignore` so you don't have to add it to every project you contribute to.

Reply:
aah nice, didn't know about that